lms/djangoapps/teams/tests/test_views.py

```diff
@@ -129,7 +129,7 @@ def test_query_counts(self):
         team.add_user(self.user)
 
         # Check the query count on the dashboard again
-        with self.assertNumQueries(19):
+        with self.assertNumQueries(20):
```

Comment:
@wajeeha-khalid A few things:
- There's no need to have an `assertNumQueries` test if it's going to be flaky (9 or 10 depending on the case) since it will make it difficult to maintain the test code.
- Instead, add the `assertNumQueries` check where it would be most reliable and useful for us to continue to verify the database calls.
- Also, since the number of queries is still going up to 20, it implies that the "select_related" call isn't working as we think.